packages/server/test/websockets.test.ts

```diff
@@ -748,7 +749,7 @@ describe('include "jsonrpc" in response if sent with message', () => {
     t.close();
   });
 
-  test('subscriptions', async () => {
+  test.only('subscriptions', async () => {
```

Comment:
This is the one that fails on my machine

```suggestion
  test('subscriptions', async () => {
```

Reply:
fixed it